qa/L0_dyna_sequence_batcher/test.sh

```diff
@@ -104,7 +104,7 @@ for i in \
         test_simple_sequence \
         test_length1_sequence \
          ; do
-    SERVER_LOG="./$i.serverlog"
+    SERVER_LOG="./$i.server.log"
```

Comment:
We can remove serverlog from Line 68. Similarly for other files.

Reply:
`rm -fr *.log *.serverlog ` becomes `rm -fr *.log *`

Reply:
adressed